src/routes/Root.tsx

```diff
@@ -14,3 +15,10 @@ export const ROOT_ROUTE: RouteObject = {
   element: <Root />,
   children: [],
 }
+
+// trying to define the dashboard as a proper route ... but im not sure this is the correct way to do it.
```

Comment:
You can remove all your route stuff. I'll fix it up lol

Reply:
thank you